### DISCUSSION TITLE
Quick help needed with Copilot Behavior (Screenshots attached)

### DISCUSSION THREAD

**Discussion by gangradem** [LazyVim/LazyVim#3306] [Q&A]
Right now, Copilot displays suggestions like this:
![image](https://github.com/LazyVim/LazyVim/assets/84750225/eaee8015-a2ad-48b5-ba8c-4e2b262a9ac9)

I want them displayed like in VS Code so I can just press tab to accept them:
![image](https://github.com/LazyVim/LazyVim/assets/84750225/f4a1016b-3275-4691-84c8-5d4295f14fdf)

By the way, I've enabled Copilot by using LazyVim's extras.

**Comment by tfriedel** [comment]
Would like to know this as well.
